Add vcpkg manifest installation instructions (#180)
snitch-org/snitch: vcpkg-example/README.md

> # Snitch example

Reviewer: ```suggestion
# Example of using *snitch* with vcpkg
```

> Snitch example that uses [vcpkg][vcpkg].

Reviewer: ```suggestion
This is an example of building a project with *snitch* as a dependency, using [vcpkg][vcpkg].
```

 
## Set up vcpkg
 

> Clone the vcpkg and run the bootstrap script.

Reviewer: ```suggestion
Clone vcpkg and run the bootstrap script.
```

 
```bash
git clone https://github.com/microsoft/vcpkg
cd vcpkg && ./bootstrap-vcpkg.bat
```
 
## Build and run the project

### CMake

> We provide a project folder called `vcpkg-example` to use `snitch`. You can build this example project according to the following code.

Reviewer: ```suggestion
The folder containing this README includes CMake files for an example empty project using  *snitch*. You can build this example project with the following code.
```
